bundles/org.openhab.binding.digiplex/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="digiplex" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<bridge-type id="bridge">
+		<label>Digiplex PRT3 module</label>
+		<description>Digiplex PRT3 module with Serial Interface</description>
+        <channels>
+            <channel typeId="messages_sent" id="messages_sent"/>
+            <channel typeId="responses_received" id="responses_received"/>
+            <channel typeId="events_received" id="events_received"/>
+        </channels>
+
+		<config-description>
+		    
```

Comment:
Formatting shows a mix of tabs and spaces might break the SAT when other users are running it.